### ISSUE TITLE
Typo @ angularjs/angular-resource.d.ts 

### ISSUE THREAD

**Issue by Skanky0815** [DefinitelyTyped/DefinitelyTyped#7187]
Please check the files for Typos.

My grunt found some issues at the angularjs/angular-resource.d.ts file.

[exec] [31m>> [39mnode_modules/DefinitelyTyped/angularjs/angular-resource.d.ts(147,49): error TS1005: ',' expected.
[exec] [31m>> [39mnode_modules/DefinitelyTyped/angularjs/angular-resource.d.ts(149,55): error TS1005: '(' expected.
[exec] [31m>> [39mnode_modules/DefinitelyTyped/angularjs/angular-resource.d.ts(150,28): error TS1005: ',' expected.

**Comment by mayankn** [comment]
I too ran into the same problem. Need a fix soon!

**Comment by vvakame** [comment]
what version of tsc do you use?
this repository only support latest tsc version.

**Comment by bartvanderwal** [comment]
`$tsc --version`
I got this same error. I was on `tsc v 1.5.3`.

I upgraded to current (tsc v1.7.3) and then it was gone.

Or actually I changed the typescript directory in the Windows PATH variable to to `C:\Program Files (x86)\Microsoft SDKs\TypeScript\1.7\`, as I already installed it previously via Visual Studio. Microsoft choose not to hack the Path variable anymore on install. Which might be a good thing, but you'll have to manage that part of version upgrades yourself now).

Don't forget to restart your command prompt before checking your version number. You might have to restart Visual Studio too (at least I had to restart Visual Studio Code, before the errors disappeared) :+1:.
